llvm/lib/Target/Hexagon/HexagonTargetMachine.cpp

```diff
@@ -220,6 +220,8 @@ LLVMInitializeHexagonTarget() {
   initializeHexagonPeepholePass(PR);
   initializeHexagonSplitConst32AndConst64Pass(PR);
   initializeHexagonVectorPrintPass(PR);
+
```

Comment:
Minor nit: Can you remove the extra line to follow the same format as the previous initialize statements?